packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2462,3 +2464,351 @@ Specific property types support a subset of [JSON Schema validations](https://to
 | `number`      | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with double-precision 64-bit IEEE 754 floating point number constraint | `minimum` and `maximum`     |
 | `integer`     | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with 32-bit signed two's complement integer constraint           | `minimum` and `maximum`     |
 | `array`       | [JSON Array](https://tools.ietf.org/html/rfc7159#section-5)                                                                         |                             |
+
+## Unique attribute
+
+<ApiLifecycle access="ea" />
+
+You can enforce uniqueness for custom properties in the Okta user profile or the Okta group profile, such as employee identification number. You can declare a maximum of 5 unique properties for each user type, and 5 unique properties in the Okta group profile. For unique properties in Okta user profile, the limit of 5 unique properties is applied per-type. You do not need to select the same set of properties for each user type. For example, the 5 custom properties you declare unique for user profile A do not need to match what you declared for user profiles B, C, or D.
+
+Unique properties in the Okta user profile share a single namespace across all [user types](/docs/reference/api/user-types) in an org. If user types A and B both contain a property ice cream and you identify it as unique in both profiles, then if user type A has the value chocolate, no other users of type A or B (or any other user type with ice cream declared unique) can have that value.
```

Comment:
```suggestion
Unique properties in the Okta user profile share a single namespace across all [user types](/docs/reference/api/user-types) in an org. If user types A and B both contain the property **ice cream** and you identify it as unique in both profiles, then if user type A has the value **chocolate**, no other users of type A or B (or any other user type with **ice cream** declared unique) can have that value.
```

Reply:
I agree with @susanharper-okta to include highlighting, but here she uses star-star to highlight the property name and value, while in the example below she suggests using back-quotes.  Which format is correct?

I also agree with her other minor edit ("the").

Reply:
Changed to ` as Susan suggested in another comment.